Library/Formula/ucommon.rb

```diff
@@ -14,16 +12,29 @@ class Ucommon < Formula
   depends_on "pkg-config" => :build
   depends_on "gnutls"
 
+  needs :cxx11
+
+  # Upstream bug report:
+  #   http://lists.gnu.org/archive/html/bug-commoncpp/2015-01/msg00000.html
```

Comment:
Oh, I usually just keep them indented on the same level. I guess it's entirely stylistic preference though.